core/build.gradle

```diff
@@ -57,4 +57,9 @@ dependencies {
     compile "io.reactivex:rxandroid:$rxAndroidVersion"
 
     compile "com.github.AntennaPod:AntennaPod-AudioPlayer:$audioPlayerVersion"
+
+    // Add casting features
+    compile "com.google.android.libraries.cast.companionlibrary:ccl:$castCompanionLibVer"
+    //compile "com.android.support:mediarouter-v7:$supportVersion"
+    //compile "com.google.android.gms:play-services:$playServicesVersion"
```

Comment:
I'm inclined to do one of two things:

1. Remove the commented out options and just depend on the dependency
2. Uncomment them and leave them in even though they'd be resolved by dependency management.